app/code/core/Mage/Eav/Model/Entity/Attribute/Backend/Abstract.php

```diff
@@ -222,7 +222,7 @@ public function getDefaultValue()
      *
      * @param Varien_Object $object
      * @throws Mage_Eav_Exception
-     * @return bool
+     * @return $this|bool
```

Comment:
I cannot see where it is returning `$this`. Can you point it out?

Reply:
It is to fix this:

```
 ------ ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- 
  Line   app/code/core/Mage/Weee/Model/Attribute/Backend/Weee/Tax.php                                                                                                                             
 ------ ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- 
  41     Return type ($this(Mage_Weee_Model_Attribute_Backend_Weee_Tax)) of method Mage_Weee_Model_Attribute_Backend_Weee_Tax::validate() should be compatible with return type (bool) of method  
         Mage_Eav_Model_Entity_Attribute_Backend_Abstract::validate()                                                                                                                             
 ------ ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- 

```

Reply:
Nasty.

Reply:
Yep. I dont like it, but i dont have a better idea?

Docs only reflect what we have to deal with ...